test/periphery/integration/merkle-campaign/lt/claim/claim.t.sol

```diff
@@ -1,263 +1,92 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.22 <0.9.0;
 
-import { Arrays } from "@openzeppelin/contracts/utils/Arrays.sol";
 import { ud2x18 } from "@prb/math/src/UD2x18.sol";
 
 import { Lockup, LockupTranched } from "src/core/types/DataTypes.sol";
-import { ISablierMerkleLT } from "src/periphery/interfaces/ISablierMerkleLT.sol";
+import { ISablierMerkleBase } from "src/periphery/interfaces/ISablierMerkleBase.sol";
 import { Errors } from "src/periphery/libraries/Errors.sol";
-import { MerkleBase, MerkleLT } from "src/periphery/types/DataTypes.sol";
+import { MerkleLT } from "src/periphery/types/DataTypes.sol";
 
-import { MerkleBuilder } from "test/utils/MerkleBuilder.sol";
-import { Merkle } from "test/utils/Murky.sol";
+import { Claim_Integration_Test } from "../../shared/claim/claim.t.sol";
 
-import { MerkleCampaign_Integration_Test } from "../../MerkleCampaign.t.sol";
-
-contract Claim_MerkleLT_Integration_Test is Merkle, MerkleCampaign_Integration_Test {
-    using MerkleBuilder for uint256[];
+contract Claim_MerkleLT_Integration_Test is Claim_Integration_Test {
```

Comment:
We can also move all of these into `MerkleLT.t.sol` and remove the `tree` from here entirely. Anyways these contract being shared cannot follow the BTT guidelines. Wdyt?

Reply:
as per current version it follows the BTT, why would it fail?

Reply:
You are right. It wouldn't make the Bulloak fail. We can keep the `tree` in `MerkleLT`.